ssl/ssl_lib.c

```diff
@@ -3889,6 +3890,22 @@ void SSL_set_not_resumable_session_callback(SSL *ssl,
                       (void (*)(void))cb);
 }
 
+void SSL_CTX_set_record_padding_callback(SSL_CTX *ctx,
+                                        size_t (*cb) (SSL *ssl,
+                                                      int type, size_t len))
+{
+    SSL_CTX_callback_ctrl(ctx, SSL_CTRL_SET_RECORD_PADDING_CB,
```

Comment:
Why go via ctrl? Why not just set it? I'm not sure ctrl is adding any value here (or for any other ssl ctrl for that matter).